etc/c/odbc/sqlucode.d

```diff
@@ -0,0 +1,472 @@
+);
+
+SQLRETURN SQLBrowseConnectW
+(
+    SQLHDBC     hdbc,
+    SQLWCHAR    *szConnStrIn,
+    SQLSMALLINT cbConnStrIn,
+    SQLWCHAR    *szConnStrOut,
+    SQLSMALLINT cbConnStrOutMax,
+    SQLSMALLINT *pcbConnStrOut
+);
+
+SQLRETURN SQLColumnPrivilegesW
+(
+    SQLHSTMT    hstmt,
+    SQLWCHAR    *szCatalogName,
+    SQLSMALLINT cbCatalogName,
+    SQLWCHAR    *szSchemaName,
+    SQLSMALLINT cbSchemaName,
+    SQLWCHAR    *szTableName,
+    SQLSMALLINT cbTableName,
+    SQLWCHAR    *szColumnName,
+    SQLSMALLINT cbColumnName
+);
+
+SQLRETURN SQLGetStmtAttrW
+(
+    SQLHSTMT   hstmt,
+    SQLINTEGER fAttribute,
+    SQLPOINTER rgbValue,
+    SQLINTEGER cbValueMax,
+    SQLINTEGER *pcbValue
+);
+
+SQLRETURN SQLSetStmtAttrW
+(
+    SQLHSTMT   hstmt,
+    SQLINTEGER fAttribute,
+    SQLPOINTER rgbValue,
+    SQLINTEGER cbValueMax
+);
+
+SQLRETURN SQLForeignKeysW
+(
+    SQLHSTMT    hstmt,
+    SQLWCHAR    *szPkCatalogName,
+    SQLSMALLINT cbPkCatalogName,
+    SQLWCHAR    *szPkSchemaName,
+    SQLSMALLINT cbPkSchemaName,
+    SQLWCHAR    *szPkTableName,
+    SQLSMALLINT cbPkTableName,
+    SQLWCHAR    *szFkCatalogName,
+    SQLSMALLINT cbFkCatalogName,
+    SQLWCHAR    *szFkSchemaName,
+    SQLSMALLINT cbFkSchemaName,
+    SQLWCHAR    *szFkTableName,
+    SQLSMALLINT cbFkTableName
+);
+
+SQLRETURN SQLNativeSqlW
+(
+    SQLHDBC    hdbc,
+    SQLWCHAR   *szSqlStrIn,
+    SQLINTEGER cbSqlStrIn,
+    SQLWCHAR   *szSqlStr,
+    SQLINTEGER cbSqlStrMax,
+    SQLINTEGER *pcbSqlStr
+);
+
+SQLRETURN SQLPrimaryKeysW
+(
+    SQLHSTMT    hstmt,
+    SQLWCHAR    *szCatalogName,
+    SQLSMALLINT cbCatalogName,
+    SQLWCHAR    *szSchemaName,
+    SQLSMALLINT cbSchemaName,
+    SQLWCHAR    *szTableName,
+    SQLSMALLINT cbTableName
+);
+
+SQLRETURN SQLProcedureColumnsW
+(
+    SQLHSTMT    hstmt,
+    SQLWCHAR    *szCatalogName,
+    SQLSMALLINT cbCatalogName,
+    SQLWCHAR    *szSchemaName,
+    SQLSMALLINT cbSchemaName,
+    SQLWCHAR    *szProcName,
+    SQLSMALLINT cbProcName,
+    SQLWCHAR    *szColumnName,
+    SQLSMALLINT cbColumnName
+);
+
+SQLRETURN SQLProceduresW
+(
+    SQLHSTMT    hstmt,
+    SQLWCHAR    *szCatalogName,
+    SQLSMALLINT cbCatalogName,
+    SQLWCHAR    *szSchemaName,
+    SQLSMALLINT cbSchemaName,
+    SQLWCHAR    *szProcName,
+    SQLSMALLINT cbProcName
+);
+
+SQLRETURN SQLTablePrivilegesW
+(
+    SQLHSTMT    hstmt,
+    SQLWCHAR    *szCatalogName,
+    SQLSMALLINT cbCatalogName,
+    SQLWCHAR    *szSchemaName,
+    SQLSMALLINT cbSchemaName,
+    SQLWCHAR    *szTableName,
+    SQLSMALLINT cbTableName
+);
+
+SQLRETURN SQLDriversW
+(
+    SQLHENV      henv,
+    SQLUSMALLINT fDirection,
+    SQLWCHAR     *szDriverDesc,
+    SQLSMALLINT  cbDriverDescMax,
+    SQLSMALLINT  *pcbDriverDesc,
+    SQLWCHAR     *szDriverAttributes,
+    SQLSMALLINT  cbDrvrAttrMax,
+    SQLSMALLINT  *pcbDrvrAttr
+);
+
+//---------------------------------------------
+// Mapping Unicode Functions
+//---------------------------------------------
+/+

```

Comment:
They're #ifdef'ed out at http://www.opensource.apple.com/source/iodbc/iodbc-36/iodbc/include/sqlucode.h?txt. I'd say keep it there as a memento and decide later whether we `version` it in upon request.